lib/util/chat/models/chat.dart

```diff
@@ -0,0 +1,64 @@
+import '../../../drives/models/drive.dart';
+import '../../../rides/models/ride.dart';
+import '../../model.dart';
+import '../../parse_helper.dart';
+import '../../supabase.dart';
+import 'message.dart';
+
+class Chat extends Model {
+  final int rideId;
+  final Ride? ride;
+
+  final int driveId;
+  final Drive? drive;
+
+  final List<Message>? messages;
+
+  Chat({
+    super.id,
+    super.createdAt,
+    required this.rideId,
+    required this.driveId,
+    this.ride,
```

Comment:
I think it would make much more sense if the ride had a chatId, instead of the other way around. That's the direction we will always need.

Reply:
i would do that in another PR if we want to start the Nutzungsstudie tomorrow

Reply:
Okay sure